CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- **Breaking**: change ``cartridge pack`` naming strategy:
+  * RPM: `<app-name>-<version>[.<suffix>]-1.<arch>.rpm`,
+  * DEB: `<app-name>_<version>[.<suffix>]-1_<arch>.deb`,
+  * TGZ: `<app-name>-<version>[.<suffix>].tar.gz`.
+  `<version>` is `git describe` output normalized to `X.Y.Z.N` or ``--version`` value.
+  (Previously it was `X.Y.Z-N-gHASH` if commits since tag and commit hash are provided,
+  `X.Y.Z-N` if only commits since tag is provided, `X.Y.Z-gHASH` if only commit hash is provided,
+  `X.Y.Z-0` if only version is provided.)
```

Comment:
How can I provide it or don't provide? I just run `cartridge pack` in a repository, not?

Reply:
You were able to pass `--version` explicitly, and this value was parsed in code

Reply:
Fixed descriptions after discussion